src/trayicon.py

```diff
@@ -332,7 +332,7 @@ def try_to_connect_to_ssid(self, ssid, ssid_info, card):
             delete_ssid_wpa_supplicant_config(ssid)
             GLib.idle_add(self.restart_authentication, ssid_info, card)
         else:
-            for _ in list(range(60)):
+            for _ in range(60):
```

Comment:
The `try_to_connect_to_ssid` method here might get a tiny speed boost because of it not eagerly growing a list before iterating.